### PR TITLE
Broken Link in the README.md.  

### PR DESCRIPTION
**Pull request by uniuuu**
Broken Link in the README.md.  

#### Release Note

```release-note

```

### CODE REVIEW

**Comment by uniuuu** [comment]
The "Try out Mattermost" link is not opening. 
Screenshot_2023-05-23_19-23-23

**Comment by M-ZubairAhmed** [comment]
@uniuuu thank you for the comment to please get started can you please sign the cla as mentioned in https://github.com/mattermost/mattermost-server/pull/23479#issuecomment-1559191929

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Other useful resources:
 Table of contents
 =================
 
-  * [Try out Mattermost](#try-out-mattermost)
+  * [Try out Mattermost](https://mattermost.com/sign-up/)
```
Comment (M-ZubairAhmed) [inline README.md:24]: This particular section title is not required anymore and can be deleted instead. It was used way back when we had a section https://github.com/mattermost/mattermost-server/tree/d8986840aca26f9e32d67ba5db9fcbf959d83338#try-out-mattermost

**Comment by cwarnermm** [comment]
@uniuuu - Are you open to applying the feedback provided in this PR and signing our CLA?

**Comment by cwarnermm** [comment]
Closing this PR due to inactivity. Please feel free to submit a new PR if needed.

**Pull request closed without merging**